mathics/builtin/numeric.py

```diff
@@ -787,3 +790,48 @@ def eval(self, x, evaluation: Evaluation):
     def eval_error(self, x, seqs, evaluation: Evaluation):
         "Sign[x_, seqs__]"
         evaluation.message("Sign", "argx", Integer(len(seqs.get_sequence()) + 1))
+
+
+class UnitStep(Builtin):
+    """
+    <url>
+    :Heaviside step function:
+    https://en.wikipedia.org/wiki/Heaviside_step_function</url> (<url>
+    :WMA link:
+    https://reference.wolfram.com/language/ref/UnitStep.html</url>)
+
+    <dl>
+      <dt>'UnitStep[$x$]'
+      <dd>return 0 if $x$ < 0, and 1 if $x$ >= 0.
+      <dt>'UnitStep[$x1$, $x2$, ...]'
+      <dd>return the multidimenional unit step function which is 1 only if none of the $xi$ are negative.
```

Comment:
typo `multidimensional`

Reply:
Thanks! Fixed now. (I should try to make codespell part of my workflow better...)